contracts/Client.sol

```diff
@@ -0,0 +1,222 @@
+      currentJob.title,
+      currentJob.employer,
+      currentJob.workAddress,
+      currentJob.startDate,
+      currentJob.endDate,
+      currentJob.monthlySalary
+    );
+  }
+
+  mapping ( uint256 => Loan ) public loanHistory;
+  uint256[] public loanIds;
+  struct Loan {
+    uint256 amount;
+    bytes32 startDate;
+    bytes32 endDate;
+    uint256 interestRate;
+    uint8 numberOfPayments;
+    bool status;
+  }
+
+  function addLoan(
+    uint256 _id,
+    uint256 _amount, 
+    bytes32 _startDate,
+    bytes32 _endDate,
+    uint256 _interestRate, 
+    uint8 _numberOfPayments, 
+    bool _status // true - success / false - success
+  ) { 
+    Loan memory newLoan = Loan(
+      _amount, 
+      _startDate, 
+      _endDate, 
+      _interestRate, 
+      _numberOfPayments, 
+      _status
+    );
+    loanIds.push(_id);
+    loanHistory[_id] = newLoan;
+  }
+
+  function getLoanRepaymentRate() constant returns (uint repaymentRate, uint loanCount) {
+    uint totalLoans = loanIds.length;
+    uint successCounter = 0;
+    for (uint i = 0; i < totalLoans; i++) {
+      if (loanHistory[i].status == true) {
+        successCounter = successCounter + 1;
+      }
+    }
+    return (successCounter, totalLoans);
+  }
+
+  function getLoan(uint256 _id) constant returns (uint256,uint256,bytes32,bytes32,uint256,uint8,bool) {
+    Loan memory currentLoan;
+    currentLoan = loanHistory[_id];
+
+    return (
+      _id, 
+      currentLoan.amount, 
+      currentLoan.startDate, 
+      currentLoan.endDate, 
+      currentLoan.interestRate, 
+      currentLoan.numberOfPayments, 
+      currentLoan.status
+    );
+  }
+
+  function Client(
+    address _clientWallet,
+    bytes32 _name,
+    bytes32 _homeAddress, 
+    bytes32 _birthday, 
+    uint256 _age, 
+    uint8 _gender, 
+    uint8 _education,
+    uint8 _householdSize,
+    uint8 _dependents,
+    bool _married,
+    uint256 _monthlyHouseholdIncome,
+    uint8 _numOfBusinesses,
+    uint8 _numOfEmployees,
+    uint256 _phoneNumber
+  ) {
+    clientInfo = PersonalInfo({
+      name: _name,
+      homeAddress: _homeAddress,
+      birthday: _birthday,
+      age: _age,
+      gender: _gender,
+      education: _education,
+      householdSize: _householdSize,
+      dependents: _dependents,
+      married: _married,
+      monthlyHouseholdIncome: _monthlyHouseholdIncome,
+      numOfBusinesses: _numOfBusinesses,
+      numOfEmployees: _numOfEmployees,
+      phoneNumber: _phoneNumber
+    });
+    clientWallet = _clientWallet;
+    numberOfJobs = 0;
+    numberOfDeposits = 0;
+  }
+
+  function getPersonalInfo() constant returns (
+    address clientWalletAddress,
+    bytes32 name, 
+    bytes32 homeAddress, 
+    bytes32 birthday, 
+    uint256 age, 
+    uint8 gender, 
+    uint8 education,
+    uint8 householdSize,
+    uint8 dependents,
+    bool married,
+    uint256 monthlyHouseholdIncome,
+    uint8 numOfBusinesses,
+    uint8 numOfEmployees,
+    uint256 phoneNumber
+  ) {
```

Comment:
Should there be permissions here?  As it is right now anyone can access anyone else's personal info, (seems like you should only allow client wallets and mifs.